src/schema/threadsByUnit.schema.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from "zod";
+
+export const threadsByUnitSchema = z.object({
+  unit_id: z.number(),
+  threads: z
+    .array(z.object({ themeSlug: z.string(), themeTitle: z.string() }))
```

Comment:
I don't suppose this could eventually be snake_case ? I'm thinking that we should try to establish the convention of DB snake_case and translate to CamelCase in the query in OWA.

Reply:
oh yeah that makes sense! this was a bit of a holdover from our previous approach so I'll update it